glean-core/ffi/src/datetime.rs

```diff
@@ -6,10 +6,13 @@ use std::os::raw::c_char;
 
 use ffi_support::FfiStr;
 
-use crate::{define_metric, handlemap_ext::HandleMapExtension, GLEAN};
+use crate::{
+    define_metric, handlemap_ext::HandleMapExtension, test_get_num_recorded_errors, GLEAN,
```

Comment:
Let's use the absolute path in the macro, then this `use` should not be needed.

(from here on I'm not gonna repeat this)